MekHQ/src/mekhq/campaign/personnel/Person.java

```diff
@@ -2103,7 +2097,7 @@ public static Person generateInstanceFromXML(Node wn, Campaign c, Version versio
                 } else if (wn2.getNodeName().equalsIgnoreCase("hits")) {
                     retVal.hits = Integer.parseInt(wn2.getTextContent());
                 } else if (wn2.getNodeName().equalsIgnoreCase("gender")) {
-                    retVal.gender = Integer.parseInt(wn2.getTextContent());
+                    retVal.gender = Gender.parseFromString(wn2.getTextContent().trim());
```

Comment:
Does parseFromString handle gender entries from older campaigns?

Reply:
It does. It defaults to valueOf but does enable integer parsing.